strapi/strapi
# Fix non required json input being set to 'null' instead of null

### What does it do?

- Fixes null being set as a string when it should be set as `null`

### Why is it needed?

- It creates a bad editing experience

### How to test it?
- Run yarn setup
- JSONInput should be empty by default
- A non-required json input should be empty by default (not `null`)
- A non-required json input can be left empty but will be set with null on save
- Emptying the json input will not fill the input with "null", but it will instead remain empty and filled with null on save and refetch.
- A required json field should be empty by default (not `{}`)
- A required json field cannot be left empty and will throw a validation error: "value is required"

### Related issue(s)/PR(s)
resolves: https://github.com/strapi/strapi/issues/15776
resolves: https://github.com/strapi/strapi/issues/15207
